packages/ladle/api/getEntryData.js

```diff
@@ -0,0 +1,3 @@
+import { getEntryData } from "../lib/cli/vite-plugin/parse/get-entry-data.js";
+
+export default getEntryData;
```

Comment:
can we import `getMetaJson` here and

```js
export default () => getMetaJson(getEntryData(config))
```

You should be able to get the config the same way as we do it `build` or `serve` commands.

1. This way, the user doesn't need to provide storyGlob again - it can be just retrieved from the same Ladle config.
2. The only new public API we are exposing is function returning `meta.json` object. `getEntryData` should stay internal - it's a complicated internal data-structure. Unless there is a strong reason to make it public, we should keep it private so we can keep changing it without worrying about breaking changes.

So the final usage should be

```js
import getMeta from @ladle/react/meta;

getMeta(); // returns JS object identical to `meta.json`
```